prac_06/guitar_test.py

```diff
@@ -0,0 +1,13 @@
+from prac_06.Guitar import Guitar
+
+
+def main():
+    guitar1 = Guitar("Gibson L-5 CES", 1922, 16035.40)
+    guitar2 = Guitar("Another Guitar", 2013, 440)
+    print("{} get_age() - Expected 99. Got {}".format(guitar1.name, guitar1.get_age()))
+    print("{} get_age() - Expected 8. Got {}".format(guitar2.name, guitar2.get_age()))
+    print("{} is_vintage() - Expected True. Got {}".format(guitar1.name, guitar1.is_vintage()))
+    print("{} is_vintage() - Expected False. Got {}".format(guitar2.name, guitar2.is_vintage()))
```

Comment:
Technically this is good, since this is a test file it can be written more efficiently.

This is what I would have done in this case:
    print(guitar1.get_age())       _# Expected 99_

Reply:
I don't see how this is more efficient, you now have to compare the output in two places instead of just on the command line, can you please elaborate?